Misc/NEWS.d/next/Library/2025-01-02-20-34-04.gh-issue-128427.onPoQZ.rst

```diff
@@ -0,0 +1,2 @@
+:data:`uuid.NIL` and :data:`uuid.MAX` are now available to represent the Nil
+and Max UUID formats as defined by :rfc:`9562`.
```

Comment:
```suggestion
:const:`uuid.NIL` and :const:`uuid.MAX` are now available to represent the Nil
and Max UUID formats as defined by :rfc:`9562`.
```

For semantic purposes, those are true constants.

Reply:
Indeed, good catch. I'd followed on from other constants that were using `:data:` just above. Can follow up with another PR to fix those.

Reply:
Oh wait. If we're already using `:data:`, keep `:data:` as well. Better to update everything in one PR if needed. We don't consistently use `const` though so I'm not really sure it's worth the hassle of a follow-up.

Reply:
It's probably just me forgetting my Sphinx 😂 

They're all defined with `.. data::`, but the few that are referenced all use `:const:`, so will follow that.

Reply:
FYI: `.. data::` is a directive while `:data:` *or* `:const:` are *roles* to reference the documented object. `const` is equivalent to `data` but can be used to distinguish semantics (`.. data::` is used for documenting both constants and non-constants module-level attributes but `:const:` is usually used to refer to module constants)